test/commands/ci/config-get-test.js

```diff
@@ -6,45 +6,88 @@ const cli = require('heroku-cli-util')
 const cmd = require('../../../commands/ci/config-get')
 
 describe('heroku ci:config:get', function () {
-  let app, coupling, key, value
+  let key, value
 
   beforeEach(function () {
     cli.mockConsole()
-    app = '123-app'
     key = 'FOO'
     value = 'bar'
+  })
+
+  context('when given an application', function () {
```

Comment:
I'd be happy for you to test the `Utils.getPipeline` and then for all of these command tests only pass a pipeline ID directly and forget about the app variant. Up to you if you want to make that change though as you've already added theses tests.